logistics_project/apps/malawi/warehouse/runner.py

```diff
@@ -236,13 +254,14 @@ def update_non_hsa_data(self, place, start, end, since, all_products=None):
                 agg = CalculatedConsumption.objects.filter(
                     update_date__gte=since,
                     supply_point__in=relevant_children,
+                    product=p
```

Comment:
@czue i thought about this a bit and came to the conclusion that product should be in this filter since we only need to re-aggregate records that have been changed recently only for this product, but let me know if you disagree